core/app/components/forms/FormElement.tsx

```diff
@@ -1,48 +1,24 @@
-import { defaultComponent } from "schemas";
```

Comment:
Refactored this to be a wrapper around `PubFieldFormElement` which now takes a lot of the inside of this function. This is because I needed to reuse the parts in `PubFieldFormElement` to render the inner value component:

![image](https://github.com/user-attachments/assets/4683cef7-5ee9-48ce-aaae-24f871992dd2)